slug.go

```diff
@@ -75,11 +75,20 @@ func packWalkFn(root, src, dst string, tarW *tar.Writer, meta *Meta, dereference
 			return nil
 		}
 
-		// Skip the .terraform directory, except for the modules subdirectory.
-		if strings.Contains(subpath, ".terraform") && info.Name() != ".terraform" {
-			if !strings.Contains(subpath, filepath.Clean(".terraform/modules")) {
-				return filepath.SkipDir
-			}
+		// Ignore the .terraform directory.
+		if info.Name() == ".terraform" {
+			return nil
+		}
+
+		// Ignore any files in the .terraform directory.
+		if !info.IsDir() && filepath.Dir(subpath) == ".terraform" {
+			return nil
+		}
+
+		// Skip .terraform subdirectories, except for the modules subdirectory.
+		if strings.Contains(subpath, ".terraform/") &&
+			!strings.Contains(subpath, filepath.Clean(".terraform/modules")) {
```

Comment:
The `filepath.Clean` is probably unnecessary since it's hardcoded, no?

Reply:
This is needed to make sure that the path separator is correct for the OS (Windows/Linux) that is being used.